commands/blacklist.js

```diff
@@ -27,6 +27,14 @@ exports.run = async (client, message, args) => {
     await client.db.updateUser(user.data)
 
     message.channel.send(`${greentick} ${client.utils.escapeMarkdown(user.tag)} (\`${user.id}\`) has been blacklisted`)
+    
+    //leave his server
+    try {
+        client.guilds.filter(c=>c.owner.id === user.id).forEach(g=>g.leave())
+    } catch (e){
+      console.log(e)
```

Comment:
Why are you using console.log to log errors?
Either do it the correct way (console.error) or don't do it at all

Reply:
Better if I remove this part

Reply:
I don't think it should be *removed*. It's always important to have callbacks
Why not just change it to the correct method?

Reply:
ok so...